src/chart/graph/GraphView.js

```diff
@@ -163,17 +164,15 @@ export default echarts.extendChartView({
 
             if (itemModel.get('focusNodeAdjacency')) {
                 el.on('mouseover', el[FOCUS_ADJACENCY] = function () {
+                    clearTimeout(this._unfocusDelayTimer);
```

Comment:
Also needs to set `this._unfocusDelayTimer` to `null` here. And in the following code with `clearTimeout`

Reply:
Thank you, I have fixed it.